Logger/TrackingEntityAttribute.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace LoggerBL
+{ 
+    public class TrackingEntityAttribute : Attribute { }
```

Comment:
This attribute could be applied for any entity. Check https://docs.microsoft.com/en-us/dotnet/api/system.attributeusageattribute?view=net-6.0